book/website/ethical-research/data-hazards/dh-how-to-use.md

```diff
@@ -0,0 +1,113 @@
+alt: A cartoon sketch of a female-presenting researcher using the Data Hazard labels on a research project. The first section in the image is the researcher looking at 10 data hazard labels under the heading 'Learn'. An arrow leads to the next step, where under the heading 'Apply' the group of people are looking at a research paper with the researcher and talking to one another, one person is raising a hand to ask a question. An arrow then leads to 'Display' where the researcher is pointing to their paper infront of an audience, with the hazard labels on each side. Learn, Apply and Display are all connected to the bigger title of Reflect, surrounded by a thought bubble. To the right-hand side of the Reflect heading, the researcher and another person are sticking hazard labels to a piece of paper.
+
+---
+_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+There are four steps to using the Data Hazard labels:
+
+1. **{ref}`Learning<er-datahazardshowtouse-labels>`**: familiarising yourself with the Data Hazard labels.
+
+2. **{ref}`Applying<er-datahazardshowtouse-apply>`**: deciding which Hazard labels are relevant to your project.
+
+3. **{ref}`Reflecting<er-datahazardshowtouse-precautions>`**: on what to do differently and what mitigations to make.
+
+4. **{ref}`Display<er-datahazardshowtouse-display>`**: displaying the labels alongside your work can help you to communicate that you've thought about these broad ethical issues and how you'd like others to use your work.
+
+In addition, whether you have used the labels yet or not, you can also **{ref}`contribute<er-datahazardshowtouse-contribute>`** to the project. If you think any labels are missing or could be improved, this project is looking to evolve with collaboration.
+
+(er-datahazardshowtouse-learning)=
+## **1. Learning** about the Data Hazard labels 
+The first step in using the Data Hazards materials is learning about the Data Hazards labels: familiarising yourself with them so that you can later {ref}`apply<er-datahazardshowtouse-apply>` them to a project.
+
+
+Data Hazards labels are supposed to represent as broad a selection of ethical risks associated with data-centric work as possible.
+This includes, but is not limited to the risks considered by  {ref}`ethics committees<er-committees>`, which often focus on risks to research participants that could lead to legal repercussions for the research organisation, such as [consent](https://datahazards.com/contents/hazards/lacks-informed-consent.html) and [privacy](https://datahazards.com/contents/hazards/risk-to-privacy.html).
+It also includes issues like [algorithmic bias](https://datahazards.com/contents/hazards/reinforces-biases.html) or [danger of misuse](https://datahazards.com/contents/hazards/danger-of-misuse.html) that might result from downstream outputs of research, rather than the research process itself.
```

Comment:
This is so cool -- I didn't know these existed, so this is definitely something I'll take with me :)